examples/colab-notebook-processing_data_using_SQl_and_local_database/colab-notebook-processing_data_using_SQl_and_local_database.ipynb

```diff
@@ -0,0 +1,387 @@
+      ],
+      "metadata": {
+        "id": "FpCoerAj0hau"
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "<a name=\"prerequisites\"></a>\n",
+        "## 1. Prerequisites"
+      ],
+      "metadata": {
+        "id": "f9XXCBpI1eMW"
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "### 1.1 Good to Know Before Your Start"
+      ],
+      "metadata": {
+        "id": "q_pbljVH2cFr"
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "This tutorial can be easily understood if you are familiar with:\n",
+        "\n",
+        "- **Python and SQL**: Basic commands and queries\n",
+        "- **Tools**: Comfort with command line and Jupyter Notebook"
+      ],
+      "metadata": {
+        "id": "jYmcEhgE2zNn"
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "### 1.2 Useful notebook shortcuts"
+      ],
+      "metadata": {
+        "id": "CKYPWV8-3XE8"
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "* Click the **Play icon** in the left gutter of the cell;\n",
+        "* Type **Cmd/Ctrl+Enter** to run the cell in place;\n",
+        "* Type **Shift+Enter** to run the cell and move focus to the next cell (adding one if none exists); or\n",
+        "* Type **Alt+Enter** to run the cell and insert a new code cell immediately below it.\n",
+        "\n",
+        "There are additional options for running some or all cells in the **Runtime** menu on top."
+      ],
+      "metadata": {
+        "id": "Mh2kLwz73mk3"
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "### 1.3 Install Versatile Data Kit and required plugins\n",
+        "\n",
+        "\n"
+      ],
+      "metadata": {
+        "id": "5s_z56EF3v3o"
+      }
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "1tN44vxJoTLa"
+      },
+      "outputs": [],
+      "source": [
+        "!pip install vdk-ipython vdk-sqlite"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "The relevant Data Job code is fetched in the sqlite-example-job directory. You can see it after running the below cell.\n",
+        "<br>Alternatively, you can use the available code as template and extend it to make a Data Job that fits your use case."
+      ],
+      "metadata": {
+        "id": "Orkqbxmq6b8V"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "!git clone https://github.com/vmware/versatile-data-kit\n",
+        "!cp -r /content/versatile-data-kit/examples/sqlite-processing-example/sqlite-example-job/ sqlite-example-job/\n",
+        "!rm -r versatile-data-kit/"
+      ],
+      "metadata": {
+        "id": "GYCT2cWkqpaL"
+      },
+      "execution_count": null,
+      "outputs": []
+    },
```

Comment:
This cell is no longer needed, correct? As the job code is inside the notebook itself now?

Though you can leave the markdwon cell with a link to https://github.com/vmware/versatile-data-kit/tree/main/examples/sqlite-processing-example/sqlite-example-job 

See alternative implementation of the same job using only python and SQL files in [here](https://github.com/vmware/versatile-data-kit/tree/main/examples/sqlite-processing-example/sqlite-example-job)